src/communication/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure CLI communication Extension #
+This is the extension for communication
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name communication
+```
+
+### Included Features ###
+#### communication communication-service ####
+##### Create #####
+```
+az communication communication-service create --name "MyCommunicationResource" --location "Global" \
+    --data-location "United States" --resource-group "MyResourceGroup" 
+
+az communication communication-service wait --created --name "{myCommunicationService}" --resource-group "{rg}"
```

Comment:
If  you want to show `wait` command usage here, please add `--no wait` in `create` command.